### ISSUE TITLE
Human meat + bun + cheese incorrectly creates human burger and not cheeseburger

### ISSUE THREAD

**Issue by Ghagliiarghii** [space-wizards/space-station-14#27941]
## Description

**Reproduction**

-   Add a slice of cheese, meat (human), and a bun to the microwave, knowing that it's supposed to cook the most complex recipe possible, in this case a cheeseburger
- Instead receive a human burger and a sad slice of cheese
- Weep

**Screenshots**

https://github.com/space-wizards/space-station-14/assets/68826635/845847d0-436c-4832-82be-a2f347c293cf

**Additional context**

**Comment by thebadman4662** [comment]
Feature. Cheese dislikes long pork and refuses to mix.

**Comment by UbaserB** [comment]
Yeah but then it invalidates another recipe, so it needs to be fixed

**Comment by themias** [comment]
There's currently no recipe for a human cheeseburger. Cheeseburger requires "raw meat," which excludes things like "raw human mean" or "raw rat meat."
